central/notifiers/README.md

```diff
@@ -0,0 +1,55 @@
+# Notifiers
+
+A notifier is used in StackRox to send notifications to a third-party system such as Splunk, Microsoft Sentinel
+or PagerDuty.
+
+## Overview
+
+Each notifier must implement the interfaces of the notifications it wants to support. Following notifier types exist.
+
+### Types of notifiers
+
+| Type                    | Interface                                  | Description                                                                                                                                   |
```

Comment:
The table is missing the `pkg/notifiers/report_notifier.go`

Reply:
done